core/kazoo_attachments/src/aws/kz_aws.erl

```diff
@@ -298,16 +299,8 @@ configure(#aws_config{} = Config) ->
 
 -spec get_metadata_credentials(aws_config()) -> {'ok', #metadata_credentials{}} | {'error', any()}.
```

Comment:
i want to keep the structure to later implement a caching with kazoo_cache, maybe reusing in kazoo_auth and/or kazoo_data and/or own kazoo_cache.